tests/UI/data/demo/countries.ts

```diff
@@ -5,47 +5,47 @@ export default {
     id: 8,
     name: 'France',
     isoCode: 'FR',
-    callPrefix: 33,
+    callPrefix: '33',
```

Comment:
Why it's not a number ?

Reply:
Because, it can contains a `+` in some cases.